lib/token.h

```diff
@@ -71,7 +78,8 @@ class CPPCHECKLIB Token {
         eNone
     };
 
-    explicit Token(Token **tokens);
+    Token();
+    explicit Token(TokensFrontBack *tokensFrontBack);
```

Comment:
hmm.. the tokensFrontBack could have the default value nullptr. Then the default constructor is not needed.